.github/workflows/tekton_bundle_push.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: Tekton Bundle Push
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['main']
+  workflow_dispatch:
+env:
+  BUNDLE: hello-world/hello-world.yaml
+  IMAGE_REGISTRY: quay.io
+  IMAGE_NAMESPACE: hacbs-release/hello-world
+  IMAGE_TAG: 0.3-alpine
+  REGISTRY_USER: ${{ secrets.QUAY_ROBOT_USER }}
+  REGISTRY_PASSWORD: ${{ secrets.QUAY_ROBOT_TOKEN }}
+jobs:
+  run-pipeline:
+    name: Tekton Bundle
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: debianmaster/actions-k3s@master
+        id: k3s
+        with:
+          version: 'latest'
+      - uses: jerop/tkn@v0.1.0
+      - uses: redhat-actions/podman-login@v1
+        with:
+          username: ${{ env.REGISTRY_USER }}
+          password: ${{ env.REGISTRY_PASSWORD }}
+          registry: ${{ env.IMAGE_REGISTRY }}
+      - run: |
```

Comment:
Can you just post a comment with an example of the output from this? For my own understanding

Reply:
Here is a link to the action:
https://github.com/redhat-actions/podman-login

Here is a screenshot from my private repo where I was developing the workflow, you can see it in action:

![image](https://user-images.githubusercontent.com/1680409/165303999-c22337fa-f493-4a02-b579-20ba842d1839.png)

Reply:
Gah, my bad @johnbieren, I probably misunderstood the question at first glance. 
You're asking about this part, right?:
```
      - run: |
          printf -v image_string '%s/%s:%s' \
            ${{ env.IMAGE_REGISTRY }} \
            ${{ env.IMAGE_NAMESPACE }} \
            ${{ env.IMAGE_TAG }} ;
          printf -v bundle_path '%s/%s' $GITHUB_WORKSPACE ${{ env.BUNDLE }} ;
          tkn bundle push -f ${bundle_path} ${image_string} ;
          tkn bundle list -o yaml --no-cache ${image_string} ;
```
Welp, just in case I've made another screen capture of the output:
![image](https://user-images.githubusercontent.com/1680409/165305783-03c81a2f-601e-4d35-9dfb-52261032a4b1.png)

The `printf` command is just formatting a string, and loading the string into an environment variable. It's a strategy to reduce long lines, as to not anger the YAML linter for lines greater-than 80 columns or whatever, and it's a neat way to set variables too.

Reply:
Yep, the latter image is what I was going for. Looks good, thanks!